apps/frontend/src/hooks/aave/useAaveReservesHistory.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * This hook is used for getting historical reserve data, and it is primarily used with charts.
+ * In particular, this hook is used in the  ̶A̶p̶y̶G̶r̶a̶p̶h̶ AaveReserveOverviewPage (chartJS).
+ */
+import { useCallback, useEffect, useState } from 'react';
+
+import dayjs from 'dayjs';
+
+import { config } from '../../constants/aave';
+import { makeCancelable } from './utils';
+
+export enum ESupportedTimeRanges {
+  OneMonth = '1m',
+  ThreeMonths = '3m',
+  SixMonths = '6m',
+  OneYear = '1y',
+  TwoYears = '2y',
+  FiveYears = '5y',
+}
+
+export const reserveRateTimeRangeOptions = [
+  ESupportedTimeRanges.OneMonth,
+  ESupportedTimeRanges.SixMonths,
+  ESupportedTimeRanges.OneYear,
+];
+export type ReserveRateTimeRange = typeof reserveRateTimeRangeOptions[number];
```

Comment:
what is this?

Reply:
it's meant to express only three options from the enum. Remember this hook was brought from https://github.com/aave/interface/blob/main/src/hooks/useReservesHistory.tsx